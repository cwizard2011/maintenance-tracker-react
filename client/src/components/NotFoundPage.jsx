import React from 'react';
import { Link } from 'react-router-dom';
import Footer from './common/Footer';

const NotFoundPage = () => (
  <div>
    <nav className="nav">
      <div className="navWide">
        <div className="wideDiv">
          <Link to="/" title="Home page"><i className="fa icon-font">&#xf015;</i></Link>
        </div>
      </div>
    </nav>
    <div className="center-heading">
      <h1>Sorry, this request page has not been created</h1>
      <p>Please click the button to start creating requests</p>
      {' '}
      <Link to="/"><button type="button" className="btn">Go home</button></Link>
    </div>
    <Footer />
  </div>
);

export default NotFoundPage;
