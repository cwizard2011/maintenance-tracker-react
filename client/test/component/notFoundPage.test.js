import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../src/components/NotFoundPage';

describe('Not found page', () => {
  it('render not found page correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
