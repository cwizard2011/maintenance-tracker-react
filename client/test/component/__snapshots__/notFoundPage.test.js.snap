// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page render not found page correctly 1`] = `
<div>
  <nav
    className="nav"
  >
    <div
      className="navWide"
    >
      <div
        className="wideDiv"
      >
        <Link
          replace={false}
          title="Home page"
          to="/"
        >
          <i
            className="fa icon-font"
          >
            ï€•
          </i>
        </Link>
      </div>
    </div>
  </nav>
  <div
    className="center-heading"
  >
    <h1>
      Sorry, this request page has not been created
    </h1>
    <p>
      Please click the button to start creating requests
    </p>
     
    <Link
      replace={false}
      to="/"
    >
      <button
        className="btn"
        type="button"
      >
        Go home
      </button>
    </Link>
  </div>
  <Footer />
</div>
`;
